src/modules/Elsa.Workflows.Runtime/Services/DefaultWorkflowRuntime.cs

```diff
@@ -90,6 +103,70 @@ public async Task<WorkflowExecutionResult> StartWorkflowAsync(string definitionI
     {
         return await StartWorkflowAsync(definitionId, options);
     }
+    
+    /// <inheritdoc />
+    public async Task CancelWorkflowAsync(string workflowInstanceId, CancellationToken cancellationToken)
+    {
+        //Should we add a cancellationtoken?
```

Comment:
I think so. After all, we might want to cancel the cancellation 😄 .

Reply:
Removed the comment as cancelling the cancellation flow could have some unwanted effects.